test-configs.yaml

```diff
@@ -1635,16 +1675,56 @@ test_configs:
       - baseline
       - kselftest
 
+  - device_type: imx6qp-sabresd
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6sll-evk
```

Comment:
No baseline results found on staging.  The devices appear to be online in the lab:
https://lavalab.nxp.com/scheduler/device_type/imx6sll-evk

but the health checks only cover the bootloader:
https://lavalab.nxp.com/scheduler/job/34088

So I don't think this can be enabled in kernelci.org yet.  What is the status?